volume/core.go

```diff
@@ -4,13 +4,13 @@ import (
 	"fmt"
 	"reflect"
 
-	"github.com/alibaba/pouch/extra/libnetwork/Godeps/_workspace/src/github.com/Sirupsen/logrus"
 	"github.com/alibaba/pouch/pkg/client"
 	metastore "github.com/alibaba/pouch/pkg/meta"
 	"github.com/alibaba/pouch/volume/driver"
 	volerr "github.com/alibaba/pouch/volume/error"
 	"github.com/alibaba/pouch/volume/types"
 	"github.com/alibaba/pouch/volume/types/meta"
+	"github.com/sirupsen/logrus"
 
 	"github.com/pkg/errors"
 	log "github.com/sirupsen/logrus"
```

Comment:
If you add the import pkg `"github.com/sirupsen/logrus"`, I think you could remove this line `"github.com/sirupsen/logrus"`, right? @faycheng

Reply:
thx, i will remove it